packages/react/src/components/ContentGroup/README.md

```diff
@@ -0,0 +1,67 @@
+# Content Group
+
+## Getting started
+
+Here's a quick example to get you started.
+
+```scss
+// yourapplication.scss
+@import '@carbon/type/scss/font-face/mono';
+@import '@carbon/type/scss/font-face/sans';
+@include carbon--font-face-mono();
+@include carbon--font-face-sans();
+@import '@carbon/ibmdotcom-styles/scss/components/content-group';
+```
+
+> 💡 Only import font's once per usage
+
+```javascript
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { ContentGroup } from '@carbon/ibmdotcom-react';
+import 'yourapplication.scss';
+
+function App() {
+  const copy = 'Lorem ipsum dolor sit amet.';
+  const type = 'heading-4';
+  return (
+    <ContentGroup heading={{ copy, type }}>
+      //Content array pattern goes here
+    </ContentGroup>
+  );
+}
+
+ReactDOM.render(<App />, document.querySelector('#app'));
+```
+
+> 💡 Don't forget to import the content group styles from
+> [@carbon/ibmdotcom-styles](https://github.com/carbon-design-system/ibm-dotcom-library/blob/master/packages/styles).
+
+## Props
+
+| Name           | Required | Data Type | Default Value                   | Description                                     |
+| -------------- | -------- | --------- | ------------------------------- | ----------------------------------------------- |
+| `heading`      | YES      | Object    | {copy: null, type: 'heading-4'} | Heading prop object                             |
+| `heading.copy` | YES      | String    | null                            | Content Group section title                     |
+| `heading.type` | NO       | String    | 'heading-4'                     | Title heading type, can be changed to heading-5 |
```

Comment:
Generally speaking, we need to align on our data modal a little better, and it's naming conventions. For example in this pull request alone we call it heading, and we call it title. Just look at lines 50-54 below here. In the functional specs I've been trying to call these titles across the board. Does heading make more sense to y'all?